Examples/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload/ViewControllers/VideoSettingsViewController.swift

```diff
@@ -90,9 +99,8 @@ class VideoSettingsViewController: UIViewController, UITextFieldDelegate
 
     private func setupAndStartOperation()
     {
-        let me = self.input!.user
-        let phAsset = (self.input!.cameraRollAsset ).phAsset
-        let operation = PHAssetCloudExportQuotaOperation(me: me, phAsset: phAsset)
```

Comment:
@nicolelehrer Yep, I just changed the name since 1) They're always `PHAssets` now (we removed `ALAsset` equivalents) and 2) This operation doesn't necessarily export from the cloud. Whether it does or not is encapsulated within apple APIs. 

I changed it to `ExportSessionExportOperation` because it first gets an `ExportSession` then uses that to `Export` the asset.

I tried to use this naming convention for all of the operations.